### PR TITLE
muon 0.1.0 (new formula)

### PR DESCRIPTION
**Pull request by meme**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/muon.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+class Muon < Formula
+  desc "Meson-compatible build system"
+  homepage "https://muon.build"
+  url "https://git.sr.ht/~lattis/muon/archive/0.1.0.tar.gz"
+  sha256 "9d3825c2d562f8f8ad96d1f02167e89c4e84973decf205d127efd9293d7da35b"
+  license "GPL-3.0-only"
+  head "https://git.sr.ht/~lattis/muon", branch: "master"
+
+  depends_on "ninja"
```
Comment (chenrui333) [inline Formula/muon.rb:9]:
```suggestion
  depends_on "ninja" => :build
```

**Review by chenrui333 — APPROVED** [review]
LGTM

File: Formula/muon.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  depends_on "pkg-config"
+
+  def install
+    system "./bootstrap.sh", "build"
```
Comment (SMillerDev) [inline Formula/muon.rb:13]: Does this download anything?

File: Formula/muon.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    system "./build/muon", "setup", "-Dprefix=#{prefix}", "build"
+    system "ninja", "-C", "build"
```
Comment (carlocab) [inline Formula/muon.rb:15]:
```suggestion
    system "./build/muon", "setup", "build", *std_meson_args
    system "ninja", "-C", "build", "--verbose"
```

File: Formula/muon.rb (hunk 2 above)
Comment (carlocab) [reply Formula/muon.rb:13]: Maybe we can bootstrap with `meson` instead?

Comment (meme) [reply Formula/muon.rb:13]: No, this script just compiles a bootstrap version of muon by compiling a single C file that performs an include of all files. The source is simple and you can view it here: https://git.sr.ht/~lattis/muon/tree/master/item/bootstrap.sh

File: Formula/muon.rb (hunk 3 above)
Comment (meme) [reply Formula/muon.rb:15]: Muon is source-compatible with Meson but the command line interface is not. For this reason, we cannot use `std_meson_args`

**Comment by meme** [comment]
Hi all. I believe the comments have been addressed.

**Comment by chenrui333** [comment]
@meme, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**